jenkins/pipelines/parallel-execution/Jenkinsfile

```diff
@@ -225,17 +232,22 @@ pipeline {
                 --product ${PRODUCT} \
                 --workspace ${PWD}
                 """
-            script {
-                //Send email for failed results.
-                if (fileExists('/testgrid/testgrid-home/jobs/${PRODUCT}/EmailReport.html')) {
-                    emailext mimeType: 'text/html',
-                            body: '${FILE,path="/testgrid/testgrid-home/jobs/${PRODUCT}/EmailReport.html"}',
-                            subject: '${PRODUCT} integration test failure!',
-                            to: 'kasung@wso2.com,pasinduj@wso2.com,yasassri@wso2.com'
-                } else {
-                  echo "No EmailReport.html file found!!"
+           script {
+                   sh """
+                   echo "/testgrid/testgrid-home/jobs/${PRODUCT}"
+                   ls -al /testgrid/testgrid-home/jobs/${PRODUCT}
+                   """
+                   //Send email for failed results.
+                   if (fileExists("/testgrid/testgrid-home/jobs/${PRODUCT}/builds/EmailReport.html")) {
+                     def emailBody = readFile '/testgrid/testgrid-home/jobs/wso2am-intg/builds/EmailReport.html'
+                     mail (to: 'kasung@wso2.com, pasinduj@wso2.com, yasassri@wso2.com, lasanthad@wso2.com, sameeraw@wso2.com, asmaj@wso2.com, harshan@wso2.com',
+                           subject: "'${env.JOB_NAME}' Integration Test Failure! #(${env.BUILD_NUMBER})",
```

Comment:
Please note that #(${env.BUILD_NUMBER}) is jenkins' build number. Testgrid's build num is different. We can keep this for now since it helps with tracking.